src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -222,17 +242,22 @@ public static boolean isGenomicsDBPath(final String path) {
             final Path featurePath = IOUtils.getPath(featureInput.getFeaturePath());
             IOUtils.assertFileIsReadable(featurePath);
             final FeatureCodec<T, ?> codec = (FeatureCodec<T, ?>) FeatureManager.getCodecForFile(featurePath, targetFeatureType);
-            return getTribbleFeatureReader(featureInput, codec);
+            return getTribbleFeatureReader(featureInput, codec, cloudWrapper, cloudIndexWrapper);
         }
     }
 
-    private static <T extends Feature> AbstractFeatureReader<T, ?> getTribbleFeatureReader(final FeatureInput<T> featureInput, final FeatureCodec<T, ?> codec) {
+    private static <T extends Feature> AbstractFeatureReader<T, ?> getTribbleFeatureReader(final FeatureInput<T> featureInput, final FeatureCodec<T, ?> codec, Function<SeekableByteChannel, SeekableByteChannel> cloudWrapper, Function<SeekableByteChannel, SeekableByteChannel> cloudIndexWrapper) {
         Utils.nonNull(codec);
         try {
             final String absolutePath = IOUtils.getPath(featureInput.getFeaturePath()).toAbsolutePath().toUri().toString();
             // Instruct the reader factory to not require an index. We will require one ourselves as soon as
             // a query by interval is attempted.
-            return AbstractFeatureReader.getFeatureReader(absolutePath, codec, false);
+
+            if (BucketUtils.isCloudStorageUrl(absolutePath)){
+                return AbstractFeatureReader.getFeatureReader(absolutePath, null, codec, false, Function.identity(), Function.identity());
+            } else {
+                return AbstractFeatureReader.getFeatureReader(absolutePath, null, codec, false, cloudWrapper, cloudIndexWrapper);
+            }
```

Comment:
Hmm, this seems backwards -- if it's a cloud storage url, you want to apply the wrappers, not the identity function. I'll fix via a push into this branch when I add the tests.